docs/io.cozy.contacts.md

```diff
@@ -25,6 +25,7 @@ The `io.cozy.contacts` doctype is loosely based on the [vCard RFC](https://tools
   - `type?`: {string} A user-provided localized type (example: `"Personnal"`, `"Profressional"`, `"Other"`)
   - `label?`: {string} A keyword to identify the type, must be `work|home`
   - `primary?`: {boolean} Indicates a preferred-use address
+- `matrix?`: {string} A simple string for the Matrix ID
```

Comment:
Could you have multiple matrix ID? In which case it should be an array
Also, I feel like this `matrix` attribute should be an object like the `cozy` attribute, in case we need to add other elements, like label&co. For the moment it could simply be `matrix:  { id: XXX }`.
Or, maybe better with a more generic approach: 
```
identifiers: [{
  service: matrix
  id: XXX
}]